main_test.py

```diff
@@ -0,0 +1,10 @@
+from os import environ
+
+environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+environ["CUDA_VISIBLE_DEVICES"] = "-1"
+
+from app.image_recognizer import ImageRecognizer # noqa
+
+image_recognizer = ImageRecognizer()
+result = image_recognizer.recognize("tests/test_data/abgr444/picture_nr_5_640x480_abgr444.raw")
+print(result)
```

Comment:
Should this file be in master branch?

Reply:
This file will be replaced with proper main by Michał in future, for now it is for demonstration purpose for him.

Reply:
[Here](https://github.com/antmicro-labs/image-recognizer/blob/color_format/run_examples.sh) is a script that would be used to run examples

Reply:
Ok, I removed it.